src/posix_disk_io.cpp

```diff
@@ -128,7 +128,7 @@ namespace {
 			storage_index_t const idx = m_free_slots.empty()
 				? m_torrents.end_index()
 				: pop(m_free_slots);
-			auto storage = std::make_shared<posix_storage>(std::move(params));
+			auto storage = std::unique_ptr<posix_storage>(new posix_storage(std::move(params)));
```

Comment:
not 100% sure if you are finally requiring c++14, but if that's the case, then `std::make_unique` here?

Reply:
oh, I thought I fixed this in a separate commit. and yes, I just switched master over to C++14 (and I may go to 17 at some point)